temporalcli/commandsmd/commands.yml

```diff
@@ -0,0 +1,2668 @@
+
+      ```
+      temporal operator namespace [command] [command options]
+      ```
+
+      For example:
+
+      ```
+      temporal operator namespace create \
+          --namespace YourNewNamespaceName
+      ```
+  - name: temporal operator namespace create
+    summary: Register a new Namespace
+    description: |
+      Create a new Namespace on the Temporal Service:
+
+      ```
+      temporal operator namespace create \
+          --namespace YourNewNamespaceName \
+          [options]
+      ````
+
+      Create a Namespace with multi-region data replication:
+
+      ```
+      temporal operator namespace create \
+          --global \
+          --namespace YourNewNamespaceName
+      ```
+
+      Configure settings like retention and Visibility Archival State as needed.
+      For example, the Visibility Archive can be set on a separate URI:
+
+      ```
+      temporal operator namespace create \
+          --retention 5d \
+          --visibility-archival-state enabled \
+          --visibility-uri YourURI \
+          --namespace YourNewNamespaceName
+      ```
+
+      Note: URI values for archival states can't be changed once enabled.
+    maximum-args: 1
+    options:
+      - name: active-cluster
+        type: string
+        description: Active Cluster (Service) name.
+      - name: cluster
+        type: string
+        description: |
+          Cluster (Service) names for Namespace creation.
+          Can be passed multiple times.
+      - name: data
+        type: string[]
+        description: |
+          Namespace data as `KEY=VALUE` pairs.
+          Keys must be identifiers, and values must be JSON values.
+          For example: 'YourKey={"your": "value"}'.
+          Can be passed multiple times.
+      - name: description
+        type: string
+        descripton: Namespace description.
+      - name: email
+        type: string
+        description: Owner email.
+      - name: global
+        type: bool
+        description: Enable multi-region data replication.
+      - name: history-archival-state
+        type: string-enum
+        description: History archival state.
+        options:
+          - disabled
+          - enabled
+        default: disabled
+      - name: history-uri
+        type: string
+        description: |
+          Archive history to this `URI`.
+          Once enabled, can't be changed.
+      - name: retention
+        type: duration
+        description: Time to preserve closed Workflows before deletion.
+        default: 72h
+      - name: visibility-archival-state
+        type: string-enum
+        description: Visibility archival state
+        options:
+          - disabled
+          - enabled
+        default: disabled
+      - name: visibility-uri
+        type: string
+        description: |
+          Archive visibility data to this `URI`.
+          Once enabled, can't be changed.
+  - name: temporal operator namespace delete
```

Comment:
I've removed the `[namespace]` part of the title and moved it into a flag. From what I've observed, this positional argument is not enforced (-y flag can be specified after the [namespace] flag, contrary to what the help doc says). 

I've also removed the same `[namespace]` part from `temporal operator namespace describe` below. In the same vein, it seems to me that we can have the same behavior with `namespace` being a flag, and with the removal of these 2, we no longer need to support the ability to specify a `[argument]` in the title, simplifying the template

Reply:
Josh has confirmed that the positional form is deprecated, safe to remove